pandas_vet/__init__.py

```diff
@@ -39,14 +48,22 @@ def check_import_name(node: ast.Import) -> List:
     return errors
 
 
+def check_inplace_false(node: ast.Import) -> List:
+    errors = []
+    for kw in node.keywords:
+        if kw.arg == "inplace" and kw.value.value is True:
```

Comment:
:trollface:  but also for real: `a = True; df.drop("foo", inplace=a)` will escape this test. Should `inplace=False` be the only way to use inplace / should we alarm on all other uses?